src/main/scala/mix/kafka/monitor/LibratoConfig.scala

```diff
@@ -0,0 +1,45 @@
+package mix.kafka.monitor
+
+import java.time.Duration
+
+case class Lag(value: Long)
```

Comment:
umm - actually if I don't use `case` I would still need to do some transformation from scala `Long` to java `Long` since `caffeine` library provides `cache` over java Generic types. 

So, I just created a case class instead rather than polluting code with java Longs and `toInstanceOf[java.lang.Long]`